intro.adoc

```diff
@@ -1,6 +1,18 @@
 [[intro]]
 == Introduction
 
-The Boot and Runtime Services specification provides the requirements for system vendors and Operating System Vendors (OSVs) to interoperate with one another by providing expectations for the Operating System (OS) to utilize in acts of device discovery, system management, and other rich operations provided in this specification. The approach taken is to leverage industry standards as well as RISC-V ratified specifications.
+The Boot and Runtime Services (BRS) specification provides the requirements for system vendors and Operating System Vendors (OSVs) to interoperate with one another by providing expectations for the Operating System (OS) to utilize in acts of device discovery, system management, and other rich operations provided in this specification. The approach taken is to leverage industry standards as well as RISC-V ratified specifications.
 
-There are two solutions initially focused on in the form of recipes that describe the various requirements needed for each. The recipe names are named BRS-I and BRS-B, interoperable and bespoke, repsectively.
+The BRS specification is targeting systems that implement HS/S/U privilege modes. This is the expected deployment for OSVs and system vendors in a typical ecosystem covering client systems up through server systems where software is provided by different vendors than the system vendor.
```

Comment:
Consider mentioning that HS mode is optional.  Something like

"The BRS specification is targeting systems that implement S/U privilege modes, and optionally the HS privilege mode."

Reply:
sg. I'll adjust.